scripts/ssb_alert.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#Check if user is passing all arguments.
+if [ "$#" != 5  ]; then
+    echo "ssb_alerts.sh provides basic daemon to fetch CERN SSB info and inject it into MONIT (AlertManager) "
+    echo "Usage: ssb_alerts.sh <ssb_query> <token-file> <alertmanager-url> <interval (in sec)> <verbosity level>"
+    exit 1
+fi
+
+query=$1
+token=$2
+alertmanager_url=$3
```

Comment:
The url, interval and verbose should have proper defaults, e.g.
```
url=${3:https//cms-monitoring.cern.ch}
interval=${4:-1}
verbose=${5:-0}
```
such that we don't need to specify them all the time. But it implies that you should properly adjust how to check number of input arguments, adjust usage string, etc.

Please adjust accordingly and test that it will work with and without defauls.

Reply:
Solved this issue.